client/src/disabledGroups.spec.ts

```diff
@@ -41,32 +47,41 @@ describe("disabledRooms", () => {
       disabledAfter: now - 1,
       name: "disabledBeforeAndAfter",
     });
+    const joinableFrom = groupFor({ joinableFrom: now - 1, name: "joinableFrom" });
+    const joinableFromAndDisabledAfter = groupFor({ disabledAfter: now -1, joinableFrom: now - 2, name: "joinableFrom" });
     const notDisabledBefore = groupFor({ disabledBefore: now - 1, name: "notDisabledBefore" });
     const notDisabledAfter = groupFor({ disabledAfter: now + 1, name: "notDisabledAfter" });
     const notDisabledBeforeAndAfter = groupFor({
       disabledBefore: now - 1,
       disabledAfter: now + 1,
       name: "notDisabledBeforeAndAfter",
     });
+    const notJoinableFrom = groupFor({ joinableFrom: now + 1, name: "notJoinableFrom" });
 
     const groups = [
       disabledBefore,
       disabledAfter,
       disabledBeforeAndAfter,
+      joinableFrom,
+      joinableFromAndDisabledAfter,
       notDisabledBefore,
       notDisabledAfter,
       notDisabledBeforeAndAfter,
+      notJoinableFrom,
     ];
 
     const result = mapPotentiallyDisabledGroups(groups);
 
     expect(result).toEqual([
-      { group: disabledBefore, isUpcoming: true, isExpired: false },
-      { group: disabledAfter, isUpcoming: false, isExpired: true },
-      { group: disabledBeforeAndAfter, isUpcoming: true, isExpired: true },
-      { group: notDisabledBefore, isUpcoming: false, isExpired: false },
-      { group: notDisabledAfter, isUpcoming: false, isExpired: false },
-      { group: notDisabledBeforeAndAfter, isUpcoming: false, isExpired: false },
+      { group: disabledBefore, isUpcoming: true, isExpired: false, isJoinable: true },
```

Comment:
That's essentially the comment from below - do we really want that isJoinable is true here?

Reply:
see above, fixed with the additonal push